hivelog_vtypes.json

```diff
@@ -0,0 +1,20 @@
+{
+    "HIVE_LOG_ENTRY": [40, {
```

Comment:
This should probably be added to the other json file. There is no need to separate them into different profiles.

Reply:
Indeed. Thanks for fixing it already!